deps.edn

```diff
@@ -8,6 +8,11 @@
         org.clojars.liverm0r/dartclojure {:mvn/version "0.2.23-SNAPSHOT"}
         vvvvalvalval/supdate {:mvn/version "0.2.3"}
         camel-snake-kebab/camel-snake-kebab {:mvn/version "0.4.3"}
+        no.cjohansen/replicant {:git/url "https://github.com/cjohansen/replicant.git"
+                                :git/sha "ef2fecbe301cafa8449b74a1e2cbe4fc3ddc18ac"}
         #_#_org.clojars.liverm0r/dartclojure {:local/root "../DartClojure"}}
+ :aliases {:dev {:extra-deps {org.clojars.abhinav/snitch {:mvn/version "0.1.14"}}}
```

Comment:
Not sure this is needed. I add it via a keykombo. But I also don't object to having it there. 😄

Reply:
Ah you mean with Calva Power Tools? That's cool.

Reply:
Even before Power Tools I had this in my custom repl commands array:

```json
      {
        "name": "qol: Add Snitch dependency",
        "repl": "clj",
        "snippet": "(require '[clojure.repl.deps :refer [add-libs]])\n\n(add-libs '{ org.clojars.abhinav/snitch {:mvn/version \"0.1.16\"}})\n\n(require '[snitch.core :refer [defn* defmethod* *fn *let]])"
      },
```

Reply:
Leaving it for now but may remove it later